src/schema/posts.ts

```diff
@@ -669,6 +669,11 @@ export const typeDefs = /* GraphQL */ `
     List of available translations for the post
     """
     translation: PostTranslation
+
+    """
+    Language of the post
+    """
+    language: String
```

Comment:
quick test (or update existing one) that it actually returns?

Reply:
Added